.azure-pipelines-templates/daily-matrix.yml

```diff
@@ -6,6 +6,8 @@ parameters:
     SGX:
       container: sgx
       pool: 1es-DC16s_v3-focal
+    SNPCC:
+      pool: sev-snp-pool
```

Comment:
Changes in this file should be reverted?

Reply:
I think it's still necessary to create .deb for SNP. Currently .deb is created only when a release is actually created, and it could potentially prevent us from make releases quickly enough.
Hopefully I can leave changes in the file but use other pool like `1es-dv4-focal`

Reply:
But if it doesn't work, I think it would be reasonable to revert code about SNP and leave Virtual as it is.

Reply:
Looks like `1es-dv4-focal` works for snp as well (because CTest was removed in past for snp).

Reply:
`1es-dv4-focal` is good here. It's OK to build the SNP release on a virtual machine.